src/localization/resourceKeys.ts

```diff
@@ -1,472 +1,472 @@
-      invokingDigitalTwinCommandWithPayload : "notifications.invokingDigitalTwinCommandWithPayload",
-      invokingMethod : "notifications.invokingMethod",
-      invokingMethodWithPayload : "notifications.invokingMethodWithPayload",
-      patchDigitalTwinInterfacePropertiesOnError : "notifications.patchDigitalTwinInterfacePropertiesOnError",
-      patchDigitalTwinInterfacePropertiesOnSuccess : "notifications.patchDigitalTwinInterfacePropertiesOnSuccess",
-      savedToIotHubConnectionString : "notifications.savedToIotHubConnectionString",
-      updateDeviceOnError : "notifications.updateDeviceOnError",
-      updateDeviceOnSucceed : "notifications.updateDeviceOnSucceed",
-      updateDeviceTwinOnError : "notifications.updateDeviceTwinOnError",
-      updateDeviceTwinOnSuccess : "notifications.updateDeviceTwinOnSuccess",
-   };
-   public static settings = {
-      cancel : "settings.cancel",
-      configuration : {
-         connectionString : {
-            label : "settings.configuration.connectionString.label",
-            link : "settings.configuration.connectionString.link",
-            sublabel : "settings.configuration.connectionString.sublabel",
-         },
-         headerText : "settings.configuration.headerText",
-      },
-      footerText : "settings.footerText",
-      headerText : "settings.headerText",
-      modelDefinitions : {
-         add : "settings.modelDefinitions.add",
-         headerText : "settings.modelDefinitions.headerText",
-         helpText : "settings.modelDefinitions.helpText",
-         repositoryTypes : {
-            device : {
-               infoText : "settings.modelDefinitions.repositoryTypes.device.infoText",
-               label : "settings.modelDefinitions.repositoryTypes.device.label",
-            },
-            private : {
-               infoText : "settings.modelDefinitions.repositoryTypes.private.infoText",
-               label : "settings.modelDefinitions.repositoryTypes.private.label",
-               textBoxLabel : "settings.modelDefinitions.repositoryTypes.private.textBoxLabel",
-            },
-            public : {
-               infoText : "settings.modelDefinitions.repositoryTypes.public.infoText",
-               label : "settings.modelDefinitions.repositoryTypes.public.label",
-            },
-         },
-      },
-      questions : {
-         headerText : "settings.questions.headerText",
-         questions : {
-            documentation : {
-               link : "settings.questions.questions.documentation.link",
-               text : "settings.questions.questions.documentation.text",
-            },
-         },
-      },
-      save : "settings.save",
-   };
-   public static template = {
-      array : {
-         add : "template.array.add",
-         remove : "template.array.remove",
-      },
-      collapse : {
-         close : "template.collapse.close",
-         open : "template.collapse.open",
-      },
-   };
+export class ResourceKeys {

```

Comment:
Anything got change in this file?

Reply:
I don't think so, I think when I build on linux the encoding or line endings change somehow